config/samples/sw.kogito_v1alpha08_kogitoserverlessworkflow_withITA.yaml

```diff
@@ -6,7 +6,8 @@ metadata:
     sw.kogito.kie.org/description: Greeting example on k8s with Italian!
     sw.kogito.kie.org/version: 0.0.2
 spec:
-  start: ChooseOnLanguage
+  start:
+    stateName: ChooseOnLanguage
```

Comment:
`start: ChooseOnLanguage` won't work anymore? I think we can have an adaptor or fix this in the SDK. 

https://github.com/serverlessworkflow/specification/blob/0.8.x/schema/workflow.json#L1812-L1839

It can be either a string or a complex object.

Reply:
Issue fixed, update coming soon.
We will rely on headless objects on the kubebuilder side and use the Admission Webhook to validate the workflow.